suite/tests/api/ir_aarch64.c

```diff
@@ -4332,6 +4367,661 @@ test_sys(void *dc)
     test_instr_encoding(dc, OP_sys, instr);
 }
 
+static int conditions[] = { DR_PRED_EQ, DR_PRED_NE, DR_PRED_CS, DR_PRED_CC,
+                            DR_PRED_MI, DR_PRED_PL, DR_PRED_VS, DR_PRED_VC,
+                            DR_PRED_HI, DR_PRED_LS, DR_PRED_GE, DR_PRED_LT,
+                            DR_PRED_GT, DR_PRED_LE, DR_PRED_AL, DR_PRED_NV };
+
+static void
+test_ccmn_imm(void *dc)
+{
+    /* ccmn <Xn>, #<imm>, #<nzcv>, <cond> */
+    for (reg_id_t rn = DR_REG_X0; rn <= DR_REG_XZR; rn++) {
+        if (rn == DR_REG_XSP) {
+            continue;
+        }
```

Comment:
Does the code format checker insist on having braces for single-line `if` statements?

Reply:
I'm not sure if code formatter insists on it. I believe using braces improves readability of code, so I'd rather keep it like this.